sources/pmwebapi.go

```diff
@@ -135,11 +137,18 @@ func newPcpSource() (PcpSource, error) {
 		if err != nil {
 			log.Fatal("NewRequest: ", err)
 		}
+		
 		for _, getValues := range timestampHeaderVar.Values {
 			for _, instanceList := range getValues.Instances {
+				labelNew := []string{"instance"}
```

Comment:
For this, I'd probably rewrite to be more like this:

labelNew := []string{}
labelValuesNew := []string{}

if instanceList.Instance != -1 {
    labelNew = append(labelNew, "instance")
    labelValuesNew = append(labelValuesNew, strconv.FormatInt(InstanceList.Instance, 10))
}